[SPI checking] Update SDKDB baseline for iOS 26

emw-apple opened the pull request:
#### a4a9126dab0ac0f128a9b586fb53bf36259905d8
<pre>
[SPI checking] Update SDKDB baseline for iOS 26
<a href="https://bugs.webkit.org/show_bug.cgi?id=299219">https://bugs.webkit.org/show_bug.cgi?id=299219</a>
<a href="https://rdar.apple.com/160975784">rdar://160975784</a>

Reviewed by Alexey Proskuryakov.

Check in a baseline generated from the iOS 26 public SDK. Remove a few
allowlist entries for staged API that ship in iOS 26.

To keep the patch size small, continue to use the &quot;iphoneos18.5&quot;
directory name, despite being technically incorrect. In a follow-up
change we will rename it to &quot;iphoneos&quot;. I think this it&apos;s appropriate to
switch to an unversioned directory name if we are going to treat the
checked-in partial SDKDBs as a baseline -- they don&apos;t need to align with
a specific SDK, they just need to represent a snapshot of the SDK that
we check against.

* Source/WebCore/Configurations/AllowedSPI-legacy.toml:
* Source/WebCore/Configurations/AllowedSPI.toml:
* Source/WebKit/Configurations/AllowedSPI-legacy.toml:
* WebKitLibraries/SDKDBs/iphoneos18.5/ARKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ARKit_Core.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ARKit_Foundation.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AVFAudio.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AVFoundation.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AVFoundation_AVFCore.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AVKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AVRouting.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Accelerate.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Accessibility.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Accounts.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AddressBook.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AppIntents.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/AssetsLibrary.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AudioToolbox.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AuthenticationServices.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AvailabilityVersions.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/BackgroundTasks.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/BrowserEngineCore.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/BrowserEngineKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/BusinessChat.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CFNetwork.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CameraCapture_AVF.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CarbonHeaders.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ClockKit.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/CloudKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Contacts.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ContactsUI.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreAudio.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreBluetooth.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreData.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreFoundation.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreGraphics.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreImage.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreLocation.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreMIDI.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreML.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreMedia.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreServices.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreSpotlight.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreTelephony.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreText.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreTransferable.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreVideo.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CryptoKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CryptoTokenKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/DataDetection.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/DeveloperToolsSupport.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/DocumentManager.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ExtensionFoundation.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ExtensionKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/FileProvider.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Foundation.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/GLKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/GameController.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/GameKit.partial.sdkdb: Removed.
* WebKitLibraries/SDKDBs/iphoneos18.5/GroupActivities.partial.sdkdb: Removed.
* WebKitLibraries/SDKDBs/iphoneos18.5/IOKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/IOSurface.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/IdentityDocumentServices.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/ImageIO.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Intents.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/LinkPresentation.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/LocalAuthentication.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/LocalAuthenticationEmbeddedUI.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ManagedAppDistribution.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/MediaAccessibility.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/MediaPlayer.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/MediaSetup.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/MediaToolbox.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Metal.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/NaturalLanguage.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Network.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/NetworkExtension.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/OSLog.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/OpenGLES.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/PDFKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/PassKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/PhotosUI.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/PrintKitUI.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/QuartzCore.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/QuickLook.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/QuickLookThumbnailing.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/RelevanceKit.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/ReplayKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/SceneKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ScreenTime.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/Security.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/SensorKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ShareSheet.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/SharedWithYou.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Social.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Speech.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/SpriteKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Symbols.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/SystemConfiguration.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/UIKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/UIKitCore.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/UIUtilities.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/UniformTypeIdentifiers.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/UserNotifications.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/VideoToolbox.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Vision.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/VisionKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/WiFiAware.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/_CoreData_CloudKit.partial.sdkdb: Removed.
* WebKitLibraries/SDKDBs/iphoneos18.5/_LocationEssentials.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/architecture.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libAppleArchive.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libAudioToolboxUtility.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libBLAS.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libBNNS.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libDER.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libQuadrature.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libSparse.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libSpatial.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libSystemHealth.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libate.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libcommonCrypto.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libcompression.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libcopyfile.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libdispatch.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libdispatch_introspection.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libdyld.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libiconv.2.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libicucore.A.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libmacho.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libobjc.A.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsqlite3.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftCore.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftCoreFoundation.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftCoreImage.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftDispatch.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftMetal.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftOSLog.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftObjectiveC.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftQuartzCore.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftUniformTypeIdentifiers.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftXPC.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswift_Volatile.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswift_math.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftos.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftsimd.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_asl.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_blocks.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_c.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_coreservices.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_dnssd.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_info.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_kernel.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_m.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_malloc.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_notify.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_platform.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_pthread.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_sandbox.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_trace.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libunwind.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libxml2.2.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libxpc.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libz.1.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/vImage.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/xnu_headers.partial.sdkdb: Renamed from WebKitLibraries/SDKDBs/iphoneos18.5/xnu.partial.sdkdb.

Canonical link: <a href="https://commits.webkit.org/300342@main">https://commits.webkit.org/300342@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/a8fe68a3549231bdfba8382848ecc4450f44fdee

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/121950 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/41652 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/32322 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/128508 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/74043 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/90f53deb-c451-459c-8bc5-bd67d815d0bf) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/123826 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/42367 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/50246 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/92675 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/61579 "Passed tests") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/72fa723e-960a-4cdc-8303-f1aefb07f682) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/124902 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/33779 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/109206 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/73332 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/41565805-eb52-4fbf-9b33-92e46db708ad) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/32789 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/27371 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/72007 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/103284 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/27564 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/131274 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/48889 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/37172 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/101238 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/49263 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/105422 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/101106 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/25689 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/46474 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/24594 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/45590 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/48746 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/54480 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/48216 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/51566 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/49896 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

aproskuryakov commented on WebKitLibraries/SDKDBs/iphoneos18.5/AVFoundation.partial.sdkdb:
```diff
@@ -2,10 +2,7 @@
 {
  "PublicSDKContentRoot": [
   {
-   "target": "arm64-apple-ios18.5"
```
Is arm64 ok to lose? I would think not.

emw-apple replied:
I believe iOS 26 dropped support for arm64-only hardware. The oldest supported SoC is the [A12](https://en.wikipedia.org/wiki/Apple_A12) on the 8th gen iPad, which was the first to support PAC.

So, system frameworks are only built for arm64e, even when arm64 software links against them.

This would be important if `audit-spi` took target triples from partial SDKDBs into account, but it doesn't—it loads declarations from all targets in the file.

aproskuryakov replied:
Is this DB only for device, not simulator?

aproskuryakov reviewed (approved):
rs=me, although with a question from spot checking.

emw-apple committed:
[SPI checking] Update SDKDB baseline for iOS 26
https://bugs.webkit.org/show_bug.cgi?id=299219
rdar://160975784

Reviewed by Alexey Proskuryakov.

Check in a baseline generated from the iOS 26 public SDK. Remove a few
allowlist entries for staged API that ship in iOS 26.

To keep the patch size small, continue to use the "iphoneos18.5"
directory name, despite being technically incorrect. In a follow-up
change we will rename it to "iphoneos". I think this it's appropriate to
switch to an unversioned directory name if we are going to treat the
checked-in partial SDKDBs as a baseline -- they don't need to align with
a specific SDK, they just need to represent a snapshot of the SDK that
we check against.

* Source/WebCore/Configurations/AllowedSPI-legacy.toml:
* Source/WebCore/Configurations/AllowedSPI.toml:
* Source/WebKit/Configurations/AllowedSPI-legacy.toml:
* WebKitLibraries/SDKDBs/iphoneos18.5/ARKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ARKit_Core.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ARKit_Foundation.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AVFAudio.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AVFoundation.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AVFoundation_AVFCore.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AVKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AVRouting.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Accelerate.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Accessibility.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Accounts.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AddressBook.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AppIntents.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/AssetsLibrary.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AudioToolbox.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AuthenticationServices.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/AvailabilityVersions.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/BackgroundTasks.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/BrowserEngineCore.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/BrowserEngineKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/BusinessChat.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CFNetwork.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CameraCapture_AVF.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CarbonHeaders.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ClockKit.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/CloudKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Contacts.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ContactsUI.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreAudio.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreBluetooth.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreData.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreFoundation.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreGraphics.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreImage.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreLocation.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreMIDI.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreML.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreMedia.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreServices.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreSpotlight.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreTelephony.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreText.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreTransferable.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CoreVideo.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CryptoKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/CryptoTokenKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/DataDetection.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/DeveloperToolsSupport.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/DocumentManager.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ExtensionFoundation.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ExtensionKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/FileProvider.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Foundation.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/GLKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/GameController.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/GameKit.partial.sdkdb: Removed.
* WebKitLibraries/SDKDBs/iphoneos18.5/GroupActivities.partial.sdkdb: Removed.
* WebKitLibraries/SDKDBs/iphoneos18.5/IOKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/IOSurface.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/IdentityDocumentServices.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/ImageIO.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Intents.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/LinkPresentation.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/LocalAuthentication.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/LocalAuthenticationEmbeddedUI.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ManagedAppDistribution.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/MediaAccessibility.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/MediaPlayer.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/MediaSetup.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/MediaToolbox.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Metal.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/NaturalLanguage.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Network.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/NetworkExtension.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/OSLog.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/OpenGLES.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/PDFKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/PassKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/PhotosUI.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/PrintKitUI.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/QuartzCore.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/QuickLook.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/QuickLookThumbnailing.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/RelevanceKit.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/ReplayKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/SceneKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ScreenTime.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/Security.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/SensorKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/ShareSheet.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/SharedWithYou.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Social.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Speech.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/SpriteKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Symbols.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/SystemConfiguration.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/UIKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/UIKitCore.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/UIUtilities.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/UniformTypeIdentifiers.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/UserNotifications.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/VideoToolbox.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/Vision.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/VisionKit.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/WiFiAware.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/_CoreData_CloudKit.partial.sdkdb: Removed.
* WebKitLibraries/SDKDBs/iphoneos18.5/_LocationEssentials.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/architecture.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libAppleArchive.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libAudioToolboxUtility.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libBLAS.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libBNNS.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libDER.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libQuadrature.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libSparse.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libSpatial.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libSystemHealth.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libate.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libcommonCrypto.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libcompression.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libcopyfile.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libdispatch.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libdispatch_introspection.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libdyld.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libiconv.2.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libicucore.A.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libmacho.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libobjc.A.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsqlite3.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftCore.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftCoreFoundation.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftCoreImage.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftDispatch.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftMetal.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftOSLog.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftObjectiveC.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftQuartzCore.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftUniformTypeIdentifiers.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftXPC.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswift_Volatile.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswift_math.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftos.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libswiftsimd.partial.sdkdb: Added.
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_asl.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_blocks.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_c.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_coreservices.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_dnssd.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_info.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_kernel.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_m.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_malloc.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_notify.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_platform.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_pthread.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_sandbox.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libsystem_trace.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libunwind.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libxml2.2.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libxpc.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/libz.1.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/vImage.partial.sdkdb:
* WebKitLibraries/SDKDBs/iphoneos18.5/xnu_headers.partial.sdkdb: Renamed from WebKitLibraries/SDKDBs/iphoneos18.5/xnu.partial.sdkdb.

Canonical link: https://commits.webkit.org/300342@main
